src/olympia/discovery/data.py

```diff
@@ -30,33 +31,33 @@ def __init__(self, *args, **kwargs):
     DiscoItem(
         addon_id=287841,
         heading=_(u'Take screenshots {start_sub_heading}with {addon_name}'
-                  '{end_sub_heading}'),
+                  u'{end_sub_heading}'),
         description=string_concat(
             '<blockquote>',
             _(u'So much more than just a screenshot tool, this add-on also '
               u'lets you edit, annotate, and share images.'),
             '</blockquote>')),
 
-    # Metal Flowers - animated
-    DiscoItem(addon_id=188332),
+    # Theme: Snow Style
+    DiscoItem(addon_id=68349, addon_name=_(u'Snow Style')),
```

Comment:
I don't think we want to translate that, since the original add-on name isn't translated.